server/src/infra/typesense-schemas/asset.schema.ts

```diff
@@ -1,6 +1,6 @@
 import { CollectionCreateSchema } from 'typesense/lib/Typesense/Collections';
 
-export const assetSchemaVersion = 9;
+export const assetSchemaVersion = 10;
```

Comment:
Making an educated guess that i need to bump this version number. Ive never worked with typesense before so please someone correct me if this is wrong!

Reply:
Yup, this looks good. I'll try to test it later.